mono/mcs/class/System.Windows.Forms/System.Windows.Forms/TextBoxBase-NSTextField.cs

```diff
@@ -170,6 +170,62 @@ public virtual void ApplyScrollbars(RichTextBoxScrollBars scrollbars)
 				// No scrollbars in this implementation
 			}
 
+			public virtual string SelectedText
+			{
+				get
+				{
+					return Text.Substring(SelectionStart, SelectionLength);
+				}
+				set
+				{
+					int selectionStart = SelectionStart;
+					Text = Text.Substring(0, selectionStart) + value + Text.Substring(selectionStart + SelectionLength);
+					Select(selectionStart + value.Length, 0);
+				}
+			}
+
+			public Color SelectionColor
+			{
+				get { 
+					NotImplemented(MethodBase.GetCurrentMethod());
```

Comment:
Since there's some implementation present it would be nice to put a comment stating why is it incomplete (or remove the `NotImplemented`).

Reply:
The true reason is that I don't know how to implement this for NSTextField. And I didn't have an example to test this with either. Having text ranges with different colors within a single-line text box seems like an unusual requirement to me. I'll add a comment.